chalicelib_smaht/check_setup.json

```diff
@@ -4,15 +4,15 @@
         "group": "Elasticsearch Checks",
         "schedule": {
             "manual_checks": {
-                "<env-name>": {
+                "data": {
                     "kwargs": {
                         "primary": true
                     }
                 }
             }
         },
         "display": [
-            "<env-name>"
+            "staging"
```

Comment:
There are a lot of these that have "data" under manual checks and "staging" under display. Is this explained somewhere? Is it right?